draft-bishop-httpbis-http2-additional-certs.md

```diff
@@ -643,16 +639,17 @@ received on any other stream MUST be rejected with a stream error of type
 ### Exported Authenticator Characteristics {#exp-auth}
 
 The Exported Authenticator API defined in [I-D.ietf-tls-exported-authenticator]
-takes as input a certificate, supporting information about the certificate
-(OCSP, SCT, etc.), and an optional `certificate_request_context`.  When
-generating exported authenticators for use with this extension, the
-`certificate_request_context` MUST be the two-octet Cert-ID.
-
-Upon receipt of a completed authenticator, an endpoint MUST check that:
-
- - the `validate` API confirms the validity of the authenticator itself
- - the `certificate_request_context` matches the Cert-ID of the frame(s) in
-   which it was received
+takes as input a request, a set of certificates, and supporting information
+about the certificate (OCSP, SCT, etc.).
+
+Upon receipt of a completed authenticator, an endpoint MUST perform the
+following steps:
+ - Using the `get context` API, retrieve the `certificate_request_context` used
+   to generate the authenticator, if any.
+ - Verify that the `certificate_request_context` is one previously generated or,
+   when processed by a client, is absent.
```

Comment:
When processed by a client that did not receive it in response to a CERTIFICATE_REQUEST?

Reply:
My intent was to say that "absent" is only allowed for clients; for either side, matching one that was previously sent is okay.  Not absent and not a previously-sent request is an invalid combination regardless.

Suggestions on better wording of that?  I don't feel like the above is any clearer that what's already written.

Reply:
I can't think of a better wording. This should be fine.